src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const sale1 = 20;
+  const sale2 = 50;
+  const duration1 = 3;
+  const duration2 = 7;
```

Comment:
Use more meaningful names of variables in future.
E.g. not 'sale1' and 'sale2' but 'smallDiscount' and 'bigDiscount' and not 'duration1' but 'shortDuration' or 'shortPeriod' etc.